python/dgllife/utils/featurizers.py

```diff
@@ -51,6 +51,7 @@
            'WeaveAtomFeaturizer',
            'PretrainAtomFeaturizer',
            'AttentiveFPAtomFeaturizer',
+           'PAGTN_AtomFeaturizer',
```

Comment:
We use camel case convention for class naming. That says, you need to remove "_".